### PR TITLE
remove param method request

### PR DESCRIPTION
**Pull request by everwhatever**
| Q               | A                                                            |
|-----------------|--------------------------------------------------------------|
| Branch?         | 1.13  |
| Bug fix?        | yes  |
| New feature?    | no |
| BC breaks?      | no |
| Deprecations?   |no |
| Related tickets | [jira ](https://sylius.atlassian.net/browse/SYL-2349) |
| License         | MIT |

### CODE REVIEW

File: src/Sylius/Bundle/ShopBundle/Resources/config/grids/product.yml [hunk 1]
```diff
@@ -15,7 +15,6 @@ sylius_grid:
                             taxon: "expr:notFoundOnNull(service('sylius.repository.taxon').findOneBySlug($slug, service('sylius.context.locale').getLocaleCode()))"
                             locale: "expr:service('sylius.context.locale').getLocaleCode()"
                             sorting: "expr:service('request_stack').getCurrentRequest().get('sorting', [])"
-                            includeAllDescendants: "expr:parameter('sylius_shop.product_grid.include_all_descendants')"
```
Comment (GSadee) [inline src/Sylius/Bundle/ShopBundle/Resources/config/grids/product.yml]: At first glance, it should not be removed, we want to have this parameter set here.

**Review by GSadee — CHANGES_REQUESTED** [review]
It seems like sth that does not resolve the initial problem from the issue: https://github.com/Sylius/Sylius/issues/14308.  You should definitely start from the scenario that reproduces the problem and then fix it properly.

**Comment by GSadee** [comment]
And as it is a bug fix, it should be opened to 1.11 branch 😃

**Comment by everwhatever** [comment]
changes are already on 1.12 branch

**Pull request closed without merging**